index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Buy My Goods</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+    <div class="page-wrap">
+    <article class="product">
+      <header>
+        <h1>Black Widow Golf Cleats</h1>
+      </header>
+      <div class="container">
+        <section class="details">
+          <img src="/images/BLACK_BW_FT.png" alt="">
+  <!--         <h3>Product Description</h3> -->
```

Comment:
go ahead and delete this line.  As a general rule of thumb comments that describe your code are good.  anytime you comment out code you can go ahead and just delete it.